src/modules/rewriter/export.ts

```diff
@@ -61,19 +61,20 @@ const handleExport = async (csvPath: string) => {
     process.exit()
   })
 
-  await Promise.each(listOfRanges.splice(counter), async ([from, to]) => {
+  for (const [from, to] of listOfRanges.splice(counter)) {
     let result: any
     try {
       result = await rewriter.exportRedirects(from, to)
     } catch (e) {
-      await saveMetainfo(metainfo, EXPORTS, indexHash, counter, listOfRoutes)
+      saveMetainfo(metainfo, EXPORTS, indexHash, counter, listOfRoutes)
```

Comment:
why you don't need the await here?

Reply:
Because `saveMetaInfo` is not an async function nor returns a promise:
```ts
export const saveMetainfo = (metainfo: any, metainfoType: string, fileHash: string, counter: number, data = {}) => {
  if (!metainfo[metainfoType]) {
    metainfo[metainfoType] = {}
  }
  metainfo[metainfoType][fileHash] = { counter, data }
  writeJsonSync(METAINFO_FILE, metainfo, { spaces: 2 })
}
```